### DISCUSSION TITLE
[Suggestions] Scenario window, the frame around the purple color of the background of the text indication of the opponents' colors can be improved

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#5444] [Ideas]
I suggest to improve the frame around the purple color of the background of the text indication of the opponents' colors.

In fheroes2 (currently):

![Scenario window 01](https://user-images.githubusercontent.com/43583994/169768844-a8b61411-f392-4e09-9183-fb1420ef4b28.png)

In fheroes2 (proposal):

![Scenario window change 04](https://user-images.githubusercontent.com/43583994/169768889-7bc54a41-d8e0-49ae-87ce-aee5d2f09f8b.png)

To compensate for the increase in vertical size of the frame, the 11 pixel vertical spaces in this issue: https://github.com/ihhub/fheroes2/issues/1250, should be reduced to 10 pixels, except for the first space at the top.

In fheroes2 (elements position adjustment):

![Scenario window, improve the visual appearance 01](https://user-images.githubusercontent.com/43583994/92324670-48eaa100-f044-11ea-9251-4dd7bc82dc94.png)

**Comment by Branikolog** [comment]
I vote for expanding this entire window and placing all the elements with more gaps, since "Necroman" had always hurt my eyes here. ;)

**Comment by LeHerosInconnu** [comment]
Hello @Branikolog,

> I vote for expanding this entire window and placing all the elements with more gaps, since "Necroman" had always hurt my eyes here. ;)

Finally some lucidity, instead of trying to fit everything into a too small window. :)
That's what I've been saying since the beginning, the window should be adapted to its content and not the other way around. :rofl:

**Comment by HaveFun91** [comment]
I'm afraid expanding of the window is not possible right now because of technical issues related to the background containing buttons etc. that cannot just move separately. I'd say we would need to prepare some background gradients etc. to have full control over the window size. Maybe someone more knowledgeable on this topic could confirm that this is the case?

**Reply by zenseii** [reply]
Yes the background gradient or pattern is the issue because it is not really repeating. This issue can be seen if you use a high resolution and go to the adventure map and view the area down to the right where resources and mini army info is found.

**Reply by HaveFun91** [reply]
That would mean that we'd have to prepare our own graphics/resources for basically everything to support any resolution. Are we allowed to do so?

**Reply by zenseii** [reply]
I believe @ihhub has a better idea of what could or should be done here.

**Comment by HaveFun91** [comment]
I just tried to make adjustment in color names, but same issue happens - borders are defined by the original graphic and there is no way to increase text color row height because of that :/
